zerver/lib/typed_endpoint.py

```diff
@@ -330,6 +330,10 @@ def parse_view_func_signature(
     "string_too_short": _("{var_name} is too short."),
     "string_type": _("{var_name} is not a string"),
     "unexpected_keyword_argument": _('Argument "{argument}" at {var_name} is unexpected'),
+    "string_pattern_mismatch": _("{var_name} does not match the required pattern"),
+    "string_fixed_length": _(
+        "{var_name} is not the correct length, correct length should be {length}"
+    ),
```

Comment:
`string_fixed_length` can be `"{var_name} is not length {length}"`; no need to fill it out with extra words.